storage: many tests failed

Many tests failed at the same time in this package.

* I will close this issue when there are no more failures in this package _and_
  there is at least one pass.
* No new issues will be filed for this package until this issue is closed.
* If there are already issues for individual test cases, I will close them when
  the corresponding test passes. You can close them earlier, if you prefer, and
  I won't reopen them while this issue is still open.

Here are the tests that failed:
* TestBucketAttrsToRawBucket (#10333)
* TestBucketAttrsToProtoBucket (#10176)
* TestCopyMissingFields (#10177)
* TestIntegration_ZeroSizedObject/http (#10334)
* TestIntegration_NonexistentObjectRead/jsonReads (#10335)
* TestIntegration_NonexistentBucket/grpc (#10336)
* TestIntegration_NoUnicodeNormalization (#10337)
* TestObjectNames
* TestObjectCompose
* TestEmptyObjectIterator
* TestEmptyBucketIterator
* TestCodecUint32
* TestUserProject
* TestRawObjectToObjectAttrs
* TestObjectAttrsToRawObject
* TestProtoObjectToObjectAttrs
* TestObjectAttrsToProtoObject
* TestAttrToFieldMapCoverage
* TestSignedURLOptionsClone
* TestErrorOnObjectsInsertCall
* TestEncryption


-----
commit: b92406ccfadfdcee379e86d6f78c901d772401a9
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/0cfa0e2b-d863-4e8c-9e0d-0e03e3c14060), [Sponge](http://sponge2/0cfa0e2b-d863-4e8c-9e0d-0e03e3c14060)
status: failed

Looks like a change to how test bucket UUIDs is generated is needed.

```--- FAIL: TestIntegration_NonexistentBucket (0.27s)
    --- PASS: TestIntegration_NonexistentBucket/http (0.23s)
    --- FAIL: TestIntegration_NonexistentBucket/grpc (0.00s)
panic: Short space called New more than 99 times. Ran out of IDs. [recovered]
	panic: Short space called New more than 99 times. Ran out of IDs.```

--- a/storage/integration_test.go
+++ b/storage/integration_test.go
@@ -214,7 +214,7 @@ func initIntegrationTest() func() error {
 }
 
 func initUIDsAndRand(t time.Time) {
-	uidSpace = uid.NewSpace("", &uid.Options{Time: t, Short: true})
+	uidSpace = uid.NewSpace("", &uid.Options{Time: t})
 	bucketName = testPrefix + uidSpace.New()
 	uidSpaceObjects = uid.NewSpace("obj", &uid.Options{Time: t})
 	grpcBucketName = grpcTestPrefix + uidSpace.New()

